src/hapi_schema/db_humanitarian_needs.py

```diff
@@ -18,15 +18,10 @@
 from hapi_schema.db_sector import DBSector
 from hapi_schema.utils.base import Base
 from hapi_schema.utils.constraints import (
-    max_age_constraint,
-    min_age_constraint,
     population_constraint,
     reference_period_constraint,
 )
 from hapi_schema.utils.enums import (
-    DisabledMarker,
```

Comment:
I think this can be deleted from the enums

Reply:
I have done so. Yeah sure is unsatisfying making data less useful.